prowler/providers/gcp/services/compute/compute_service.py

```diff
@@ -365,6 +374,9 @@ class Instance(BaseModel):
     service_accounts: list
     ip_forward: bool
     disks_encryption: list
+    automatic_restart: bool = True
```

Comment:
I would initialize the attribute with the non compliant value. If by any chance we can't retrieve it, is better to give a false positive than a false negative in my opinion.
```suggestion
    automatic_restart: bool = False
```